<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalcCreditView</class>
 <widget class="QWidget" name="CalcCreditView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>361</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="4">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>25</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Кредитный калькулятор</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_">
     <property name="text">
      <string>Общая сумма кредита:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2" colspan="3">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Ежимесячный платеж</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLineEdit" name="lineEdit_all_sum">
     <property name="styleSheet">
      <string notr="true"> QLineEdit {
    border: 2px solid gray;
    border-radius: 10px;
    padding: 0 8px;
    background: white;
    selection-background-color: darkgray;
       background-color: rgba(230, 228, 232, 199);
}</string>
     </property>
     <property name="maxLength">
      <number>20</number>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="label_12">
     <property name="text">
      <string>руб.</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLabel" name="label_all_sum">
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QLabel" name="label_point">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QLabel" name="label_mouth">
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Срок кредита</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2" colspan="3">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Переплата по кредиту </string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLineEdit" name="lineEdit_credit_term">
     <property name="styleSheet">
      <string notr="true"> QLineEdit {
    border: 2px solid gray;
    border-radius: 10px;
    padding: 0 8px;
    background: white;
    selection-background-color: darkgray;
       background-color: rgba(230, 228, 232, 199);
}</string>
     </property>
     <property name="maxLength">
      <number>10</number>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLabel" name="label_13">
     <property name="text">
      <string>мес.</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QLabel" name="label_credit_term">
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Процентная ставка</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2" colspan="2">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Общая выплота</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLineEdit" name="lineEdit_interest_rate">
     <property name="styleSheet">
      <string notr="true"> QLineEdit {
    border: 2px solid gray;
    border-radius: 10px;
    padding: 0 8px;
    background: white;
    selection-background-color: darkgray;
       background-color: rgba(230, 228, 232, 199);
}</string>
     </property>
     <property name="maxLength">
      <number>4</number>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLabel" name="label_14">
     <property name="text">
      <string>%</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QLabel" name="label_interest_rate">
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Тип</string>
     </property>
    </widget>
   </item>
   <item row="7" column="2" colspan="2">
    <widget class="QPushButton" name="pushButton_result">
     <property name="styleSheet">
      <string notr="true">QPushButton {
 background-color: rgba(117, 116, 118, 181)
 border-radius: 12px;
 color: white;
 border: 1px solid gray;
}

QPushButton:pressed {
   background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                     stop: 0 #B22222, stop: 1 #B22222);
}</string>
     </property>
     <property name="text">
      <string>Расчитать</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="2">
    <widget class="QComboBox" name="comboBox_type">
     <property name="styleSheet">
      <string notr="true"> QComboBox {
    border: 1px solid gray;
    border-radius: 3px;
    padding: 1px 18px 1px 3px;
    min-width: 6em;
}

QComboBox:editable {
    background: white;
}

QComboBox:!editable, QComboBox::drop-down:editable {
     background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0 #E1E1E1, stop: 0.4 #DDDDDD,
                                 stop: 0.5 #D8D8D8, stop: 1.0 #D3D3D3);
}

/* QComboBox получает состояние &quot;on&quot;, когда всплывающий список раскрыт */
QComboBox:!editable:on, QComboBox::drop-down:editable:on {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                stop: 0 #D3D3D3, stop: 0.4 #D8D8D8,
                                stop: 0.5 #DDDDDD, stop: 1.0 #E1E1E1);
}

</string>
     </property>
     <item>
      <property name="text">
       <string>Аннуитетный</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Дифференцированный</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="8" column="2" colspan="3">
    <widget class="QPushButton" name="pushButton_close">
     <property name="text">
      <string>Закрыть окно</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
